src/codegen/templates/task/task.py.mako

```diff
@@ -242,9 +251,9 @@ def export_signals(self):
         return self._export_signals
 
     @property
-    def in_stream(self):
+    def in_stream(self) -> InStream: 
         """
-        :class:`nidaqmx._task_modules.in_stream.InStream`: Gets the read
+        :class:`nidaqmx.task._in_stream.InStream`: Gets the read
```

Comment:
I don't like how this codebase has been using private types in docstrings up to now.

`_in_stream` is currently a private submodule, so docstrings should use the alias: `:class:``nidaqmx.task.InStream``` (ignore markdown mess) in docstrings. This might not work until you set `__all__`.

If you make `nidaqmx.task.in_stream` public, then it's ok to use it in docstrings, but that makes the docstrings more verbose.